styles/components/factsheets/area-select.scss

```diff
@@ -31,10 +31,22 @@
       }
 
       .value-information {
+        display: flex;
+        align-items: center;
         margin-top: 7px;
         font-family: $font-family-1;
         font-size: $font-size-medium;
         letter-spacing: -1px;
+
+        .star-icon {
+          width: 16px;
```

Comment:
Icon sizes are setted in `icons` styles file. If you need to positionate them or similar do it here.